### PR TITLE
Fix for stripline code

### PR DESCRIPTION
**Pull request by andresmmera**
This commit fixes some issues detected during the test of PR#510:

* The synthesis procedure is refined using the analysis formulae
* Removed redefinition of C0

### CODE REVIEW

File: qucs/qucs-transcalc/stripline.cpp [hunk 1]
```diff
@@ -167,9 +176,29 @@ int stripline::synthesize ()
...
   double A2 = (x/(pi*(1-x))) * (1 - 0.5*log( (x*x/(4-2*x+x*x)) + pow((0.0796*x/( (We/(2*h)) + 1.1*x)),m) ));
 
-  double W_ = (A1 - A2)*(2*h-t);
-
-  setProperty ("W", W_, UNIT_LENGTH, LENGTH_M);
+  double Wi = (A1 - A2)*(2*h-t);//Width given by the zero-thickness approximation (Initial guess for the refinement)
+ 
+  double Zi, diff_Zi, Zi__;
+  double dW = Wi*1e-4;//Differential width
+  double err = fabs(Zi - Z0);//Error metric: Norm-1
```
Comment (guitorri) [inline qucs/qucs-transcalc/stripline.cpp]: Why not use std::abs?

File: qucs/qucs-transcalc/stripline.cpp [hunk 2]
```diff
@@ -167,9 +176,29 @@ int stripline::synthesize ()
...
+  unsigned int MAX_ITER = 100, iter = 0;
+  double MAX_ERR = 1e-5;
+  double slope, Zi_1 = 0, Zi_diff, step;
+
+
+  // The Newton-Raphson method is employed to refine the width given by the zero-thickness approximation with the analysis formulae.
+  while ((fabs(Z0 - Zi_1) > MAX_ERR)&&(iter < MAX_ITER))//Stop condition: |Z0 - Z_i| < MAX_ERR or max. number iterations exceeded
```
Comment (guitorri) [inline qucs/qucs-transcalc/stripline.cpp]: std::abs?

**Comment by andresmmera** [comment]
I've used _fabs()_ in the stripline code because it is used in the following places:

https://github.com/Qucs/qucs/blob/master/qucs/qucs-transcalc/c_microstrip.cpp#L429
https://github.com/Qucs/qucs/blob/master/qucs/qucs-transcalc/c_microstrip.cpp#L667
https://github.com/Qucs/qucs/blob/master/qucs/qucs-transcalc/coplanar.cpp#L212
https://github.com/Qucs/qucs/blob/master/qucs/qucs-transcalc/coplanar.cpp#L245
https://github.com/Qucs/qucs/blob/master/qucs/qucs-transcalc/microstrip.cpp#L534
https://github.com/Qucs/qucs/blob/master/qucs/qucs-transcalc/microstrip.cpp#L555

Should I replace these _fabs()_ by _std::abs()_?

**Comment by guitorri** [comment]
My bad, I though qucs-transcalc was included in #655. 
If you want to do std::abs() please do, perhaps in another PR.

**Comment by andresmmera** [comment]
Uhm, should I wait until someone merges #667 to open that PR?

**Comment by andresmmera** [comment]
"grep -IR fabs" shows (a lot of) matches. 
Should we replace fabs() -> std::abs() everywhere?

You can see the output here https://pastebin.com/VvYZSgku

**Pull request merged**